### ISSUE TITLE
Ручная загрузка отчета по версиям из mxl не загружает даты версий, если отчет был сформирован в базе с отличной от русской локализацией

### ISSUE THREAD

**Issue by leemuar** [1C-Company/GitConverter#57]
Если создать вручную отчет по хранилищу в базе, локализация которой отличается от "русский"
И загрузить отчет по хранилищу в Конвертер
Тогда у версий будет отсутствовать дата создания

![image](https://user-images.githubusercontent.com/5869791/47292610-33b32780-d610-11e8-8868-e8b75cf63c44.png)

Английская локализация:
![image](https://user-images.githubusercontent.com/5869791/47292656-63fac600-d610-11e8-8edf-067363142e08.png)

Русская локализация:
![image](https://user-images.githubusercontent.com/5869791/47292800-ceac0180-d610-11e8-8c44-79054d436d61.png)

Судя по всему алгоритм чтения версий заточен только под российский формат даты/времени. Это нормально работает при автоматической загрузке отчета т.к. там используются флаги ```/L ru /VL ru```, а вот ручная просто "молча" создает версии с пустыми датами.

https://github.com/1C-Company/GitConverter/blob/6f100e3/GitConverter/src/Catalogs/%D0%92%D0%B5%D1%80%D1%81%D0%B8%D0%B8%D0%A5%D1%80%D0%B0%D0%BD%D0%B8%D0%BB%D0%B8%D1%89%D0%B0/ManagerModule.bsl#L220-L224

**Comment by marmyshev** [comment]
Да, надо переключить локализацию на русский  в настройках запуска Стартера - добавить эти ключи при запуске.

**Comment by leemuar** [comment]
Предлагаю либо явно сообщать о невозможности преобразования даты при ручной загрузке либо научить ручную загрузку читать даты произвольных локализаций

**Comment by leemuar** [comment]
взял в работу

Пока не уверен смогу ли сделать конвертацию даты из любых форматов, но считаю, что точно нужно выдавать ошибку, если прочитать дату не удалось, и НЕ загружать из файла, если хотя бы у одной версии не удалось прочитать дату
